utils/src/api_schema/api_schema.json

```diff
@@ -664,6 +694,20 @@
         "unique":      { "type": "boolean" }
       },
 
+      "additionalProperties": false
+    },
+
+    "FindFrames": {
+      "properties": {
+        "frames":      { "$ref": "#/definitions/nonNegativeIntArray" },
+        "_ref":        { "$ref": "#/definitions/refInt" },
+        "link":        { "$ref": "#/definitions/blockLink" },
```

Comment:
Is there a wiki page for this? Its good to make this definition consistent with the wiki and with other similar commands

Reply:
https://github.com/IntelLabs/vdms/wiki/FindFrames